src/test/java/view/HTMLReportBuilderTest.java

```diff
@@ -88,15 +91,14 @@ static void beforeAll() {
     }
 
     @Test
-    void createHTMLReport() {
-        try {
-            Path path = Files.createTempDirectory("html");
-            HTMLReportBuilder.createHTMLReport(packages, path.toString() + "/");
-            String result = IOUtil.readFile(path.toString() + "/" + "changelog_" + java.time.LocalDate.now() + ".html").replace(java.time.LocalDate.now().toString(), "");
-            String expected = IOUtil.readFile("src/test/resources/html/" + "changelog_expected.html");
-            assertThat(result).isEqualToIgnoringWhitespace(expected);
-        } catch (IOException e) {
-            LOGGER.error(e.getMessage());
-        }
+    void createHTMLReport() throws IOException {
+        Path path = Files.createTempDirectory("html");
+        HTMLReportBuilder.createHTMLReport(packages, path.toString());
+        File resultFile = Objects.requireNonNull(new File(path.toString()).listFiles())[0];
+
+        String result = IOUtil.readFile(resultFile.getPath()).replace(LocalDate.now().toString(), "");
```

Comment:
Immer noch LocalDate asynchronität failure problematik.
```suggestion
        String result = IOUtil.readFile(resultFile.getPath()).replaceFirst("Dependency changelog from .+", "Dependency changelog from");
```

Reply:
Gute Lösung implementiert in 5958b7552a3e45a4aad76bf311a6b262e67b6eb9